app/controllers/boxes.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from flask import Blueprint, jsonify
+
+from app import db
+from app.schemas.boxes import BoxSchema
+from app.schemas.votes import VoteSchema
+from app.models.boxes import BoxModel
+from app.models.options import OptionModel
+from app.models.votes import VoteModel
+from app.helpers import validate_input, token_required
+
+boxes_blueprint = Blueprint("boxes", __name__)
+
+
+@boxes_blueprint.route("/boxes", methods=["GET"])
+@token_required
+def get_all_boxes(user_id):
+    list_boxes = db.session.query(BoxModel).filter(BoxModel.user_id == user_id).all()
+
+    return jsonify(BoxSchema(many=True).dump(list_boxes)), 200
+
+
+@boxes_blueprint.route("/boxes", methods=["POST"])
+@token_required
+@validate_input(schema=BoxSchema)
+def create_new_box(user_id, data):
+    name = data['name']
+    description = data['description']
```

Comment:
I thought about this! ```alnum()``` seems to be a good validation if we want to allow only alphabet and number. However, I think name and description of the box depend on the owner. It will be False if I have input "minh." due to the dot, this is bad as well. Hence, I decided to keep the current version.